tests/unit9.test.js

```diff
@@ -0,0 +1,60 @@
+const { test, expect } = require('@playwright/test');
+
+test('Perform Login:', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.getByPlaceholder('Username').fill('standard_user');
+  await page.getByPlaceholder('Password').fill('secret_sauce');
+  await page.locator('input[data-test="login-button"]').click();
```

Comment:
Please read this doc https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute. if you make changes in PW config file then you can get elements like `await page.getByTestId('login-button').click();`
Please use this approach for test below too